src/Pages/Login.js

```diff
@@ -1,5 +1,65 @@
+import {Box, Button, TextField, Typography} from '@mui/material';
+import {useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {Navigate} from 'react-router-dom';
+import {addUser} from '../redux/redux-reducers/userstore';
+
 const Login = () => {
-  return <div>LOGIN</div>;
+  const loggedIn = useSelector((state) => state.userStore.loggedIn);
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState('');
+  const handleClick = () => {
+    //checking if it's an email
+    if (email.includes('@')) {
+      dispatch(addUser(email));
+      window.location.reload();
+    } else {
+      alert('Please enter a valid e-mail');
```

Comment:
Please use toast for this. Let @Dzastinche set up the store for toast first.
https://www.npmjs.com/package/react-toastify

Reply:
I will wait for him to set up the store